packages/frontend/apps/electron/src/main/application-menu/create.ts

```diff
@@ -171,22 +171,6 @@ export function createApplicationMenu() {
             switchToPreviousTab();
           },
         },
-        {
-          label: 'Switch to next tab (mac)',
-          accelerator: 'Command+]',
-          visible: false,
-          click: () => {
-            switchToNextTab();
-          },
-        },
-        {
-          label: 'Switch to previous tab (mac)',
-          accelerator: 'Command+[',
-          visible: false,
-          click: () => {
-            switchToPreviousTab();
-          },
-        },
         {
           label: 'Switch to next tab (mac 2)',
```

Comment:
do we need to relabel this to `(mac)` ?

Reply:
this is hidden and the label is just for debugging purpose